assets/javascripts/discourse/controllers/group-reports-show.js.es6

```diff
@@ -0,0 +1,46 @@
+import Query from "discourse/plugins/discourse-data-explorer/discourse/models/query";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { ajax } from "discourse/lib/ajax";
+import {
+  default as computed,
+  observes
+} from "ember-addons/ember-computed-decorators";
+
+export default Ember.Controller.extend({
+  showResults: false,
+  explain: false,
+  loading: false,
+  results: Ember.computed.alias("model.results"),
+  hasParams: Ember.computed.gt("model.param_info.length", 0),
+
+  actions: {
+    run() {
+      this.setProperties({ loading: true, showResults: false });
+      ajax(`/g/${this.get("group.name")}/reports/${this.get("model.id")}/run`, {
+        type: "POST",
+        data: {
+          params: JSON.stringify(this.get("model.params")),
+          explain: this.explain
+        }
+      })
+        .then(result => {
+          this.set("results", result);
+          if (!result.success) {
+            this.set("showResults", false);
+            return;
+          }
+
+          this.set("showResults", true);
```

Comment:
a lot of `set` here, I don't think we need this anymore `this.showResults = false` should do